js/annotationtoolbar.mjs

```diff
@@ -144,6 +169,15 @@ export class AnnotationToolbar{
     hide(){
         $(this.ui.buttongroup.element).hide();
     }
+    
+/**
+ * Adds the Annotation Toolbar to an OpenSeadragon viewer.
+ *
+ * @memberof AnnotationToolbar
+ * @instance
+ * @function addToOpenSeadragon
+ * @param {Object} viewer - The OpenSeadragon viewer.
+ */
```

Comment:
Additional information about how and where the toolbar is added would be useful (i.e. an OpenSeadragon.ButtonGroup is created and added to the OpenSeadragon.ControlAnchor.TOP_LEFT position). Also mention that the .viewer-controls-topleft class is added.